implementation.js

```diff
@@ -157,6 +158,8 @@ module.exports = function from(arrayLike) {
 	if (arrayFromIterable) {
 		items = arrayFromIterable;
 		len = arrayFromIterable.length;
+	} else if (isString(items)) {
+		return items.match(/[\uD800-\uDBFF][\uDC00-\uDFFF]?|[^\uD800-\uDFFF]|./g) || [];
```

Comment:
To be extra robust, we should cache `String.prototype.match` at the module level, and `match.call(items, regex)` here.